$OpenBSD: patch-libsrc_Wi_recovery_c,v 1.1.1.1 2013/04/22 17:32:31 zhuk Exp $
Replace hand-crafted zeroing logic with simplier and non-warning one.
--- libsrc/Wi/recovery.c.orig	Tue Jan 29 20:25:38 2013
+++ libsrc/Wi/recovery.c	Tue Jan 29 20:27:56 2013
@@ -396,8 +396,8 @@ bkp_check_and_recover_blob_cols (it_cursor_t * itc, db
 		      log_error ("will have to set blob for col %s in key %s to empty",
 				 col_name, key->key_name);
 
-		      INT64_SET_NA (col + BL_CHAR_LEN, 0L);
-		      INT64_SET_NA (col + BL_BYTE_LEN, 0L);
+		      memset(col + BL_CHAR_LEN, 0, sizeof(int64));
+		      memset(col + BL_BYTE_LEN, 0, sizeof(int64));
 		      updated = 1;
 		    }
 		}
